### ISSUE TITLE
Allow to customize fzf and bat UI

### ISSUE THREAD

**Issue by petobens** [iloveitaly/ipython-ctrlr-fzf#4]
Hi! Great ipython extension! I was wondering whether you could add the possibility of customizing both the fzf ui and the bat look and feel. 

For instance the bat config seems to be hardcoded here https://github.com/iloveitaly/ipython-ctrlr-fzf/blob/51d6a3cd4aeb62a73151df9225a15c2693d11bf0/ipython_ctrlr_fzf/__init__.py#L40 and I would like to pass a theme flag to it `--theme TwoDark`.  Similarly for fzf I would like to pass some options as in https://github.com/petobens/dotfiles/blob/69ab3ed83203f25b5e36dd97964311ad45cf6dae/fzf_bash.sh#L34. 

Thanks in advance!

**Comment by iloveitaly** [comment]
Definitely, that sounds like an awesome idea. Want to submit a PR adding that?

**Comment by petobens** [comment]
Sure. Dunno though how you want me to do it: should it read an env var? Or is is possible to pass options to the extension upon loading?

**Comment by petobens** [comment]
In #8 I added the ability to read fzf default opts from an env variable. For bat this is not needed since we can simply add settings in a config file  and the bat executable will pick that up.
